packages/internal/src/node/createBundles.js

```diff
@@ -23,7 +23,7 @@ export const createBundlesFromAbsolute = async sourceBundles => {
     }
     const bundle = match[1];
 
-    const args = cacheToArgs.get(cache) || ['--to', cache];
+    const args = cacheToArgs.get(cache) || ['--cache-js', cache];
```

Comment:
Is this assuming we have a newer endo version than we actually have?

Reply:
I have verified that this change was added in https://github.com/endojs/endo/pull/1278 which first landed in `@endo/bundle-source@2.3.0`, and the release branch is based of `@endo/bundle-source@2.5.2`.